tests/js/JSStandardBearTest.py

```diff
@@ -0,0 +1,77 @@
+from bears.js.JSStandardBear import JSStandardBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
```

Comment:
The imports should be like : 
```
from coalib.testing.LocalBearTestHelper import verify_local_bear

from bears.js.JSStandardBear import JSStandardBear
```
I know that many files have imports in the order that you've done but that's incorrect. See PEP8 style of imports :wink: